### PR TITLE
core(lhr): s/audits/auditRefs, s/category.name/category.title/

### PR DESCRIPTION
**Pull request by patrickhulce**
ref #4333 #5008

### CODE REVIEW

File: typings/lhr.d.ts [hunk 1]
```diff
@@ -41,15 +41,15 @@ declare global {
...
         /** A description for the manual audits in the category. */
         manualDescription?: string;
         /** The overall score of the category, the weighted average of all its audits. */
         score: number|null;
         /** An array of references to all the audit members of this category. */
-        audits: CategoryMember[];
+        auditRefs: CategoryMember[];
       }
 
       export interface CategoryMember {
```
Comment (brendankenny) [inline typings/lhr.d.ts]: `AuditRef`

File: typings/config.d.ts [hunk 2]
```diff
@@ -103,7 +103,7 @@ declare global {
       // TODO: For now, these are unchanged from JSON and Result versions. Need to harmonize.
       export interface CategoryMember extends CategoryMemberJson {}
       export interface Category extends CategoryJson {
-        audits: CategoryMember[];
```
Comment (brendankenny) [inline typings/config.d.ts]: `auditRef`/`auditRefJson` up here too

File: typings/lhr.d.ts (hunk 1 above)
Comment (patrickhulce) [reply typings/lhr.d.ts]: done

File: typings/config.d.ts (hunk 2 above)
Comment (patrickhulce) [reply typings/config.d.ts]: done

**Pull request merged**